src/js/providers/html5.js

```diff
@@ -368,6 +369,9 @@ define([
                 _setVideotagSource();
                 _videotag.load();
             } else {
+                if (_forceStateToLoad) {
+                    _this.setState(states.LOADING);
```

Comment:
Let us try to put the setState(loading) in the two places where completeLoad is called. For quality level don't check for ismobile.

In the future, isMobile should be "is initiated by a touch event on mobile, or is desktop"